decrypt/decrypt.go

```diff
@@ -90,16 +94,35 @@ func Decrypt(args []string) error {
 			err := decryptFile(file.Encrypted, file.Unencrypted, *privateKey)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "Error decrypting file %s: %v\n", file.Encrypted, err)
+				decryptedCount--
 			}
+			decryptedCount++
 		case helpers.FileExists(file.Unencrypted):
 			fmt.Fprintf(os.Stderr, "Warning: file %s is already decrypted, skipping\n", file.Unencrypted)
 		default:
 			fmt.Printf("Decrypting file %v/%v: %s\n", i+1, numFiles, file.Encrypted)
 			err := decryptFile(file.Encrypted, file.Unencrypted, *privateKey)
 			if err != nil {
 				fmt.Fprintf(os.Stderr, "Error decrypting file %s: %v\n", file.Encrypted, err)
+				decryptedCount--
 			}
+			decryptedCount++
 		}
+		// remove the encrypted file if the clean flag is set
+		if *clean && helpers.FileIsReadable(file.Encrypted) {
```

Comment:
```suggestion
		if *clean {
```

The start of the switch statement assures that the file is readable